osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -412,6 +414,7 @@ public enum OsuSetting
         EditorLimitedDistanceSnap,
         ReplaySettingsOverlay,
         AutomaticallyDownloadMissingBeatmaps,
-        EditorShowSpeedChanges
+        EditorShowSpeedChanges,
+        GameplayDisableTaps,
```

Comment:
I would want to include `Touch` anywhere in this setting name so I can easily find it, "Gameplay" sounds like a broad term to me for what is meant to be a touch-specific setting.

Reply:
`TouchDisableGameplayTaps`